scripts/script.js

```diff
@@ -60,48 +60,109 @@ export default function () {
   }
 }
 
+/**
+ * Fetches a piece CID from the BOOST_FETCH_URL
+ * @param {string} piece The piece CID string to fetch
+ * @returns A K6 HTTP response from the BOOST_FETCH_URL (https://k6.io/docs/javascript-api/k6-http/response/)
+ */
 function fetchFromBoost(piece) {
-  let response = http.get(`${__ENV.BOOST_FETCH_URL}${piece}`, {
+  let response = get(`${__ENV.BOOST_FETCH_URL}${piece}`, {
     tags: {
       name: 'BoostFetchURL',
-    },
-    timeout: `${__ENV.SIMULTANEOUS_DOWNLOADS}h`,
+    }
   })
   timeBoost.add(response.timings.duration)
   ttfbBoost.add(response.timings.waiting)
-  boostSuccess.add(
-    response.status >= 200 && response.status < 300
-  )
+  boostSuccess.add(response.status >= 200 && response.status < 300)
 
-  let contentLength = parseInt(response.headers['Content-Length'])
-  dataReceivedBoost.add(contentLength, { url: response.url })
-  bytesPerMsBoost.add(contentLength / response.timings.duration)
+  if (response.headers["Content-Length"] !== undefined) {
```

Comment:
nit: mixing quote styles
```suggestion
  if (response.headers['Content-Length'] !== undefined) {
```